docs/src/lucet-runtime/killswitch.md

```diff
@@ -0,0 +1,341 @@
+# `KillSwitch`
+
+`KillSwitch` is a mechanism by which users of `lucet-runtime` can
+asynchronously request, very sternly, that an instance be disabled from
```

Comment:
It feels like phrase "very sternly" -- as well as the specificity of the subsequent sentences -- suggests that there are scenarios in which one can request a KillSwitch but it doesn't have the intended effect.  If that's the case, would it be worthwhile to briefly introduce error scenarios.

Reply:
Yes, definitely! I hint later on at the implication made here, but this should at least directly reference that related reading.